.gitignore

```diff
@@ -0,0 +1,18 @@
+# macOS artifacts
+.DS_Store
+
+# IDEs / editors
+.vscode/
+*~
+
+# Common cmake build folders
+build/
+build-*
+
+# Usually you don't want to commit binaries
```

Comment:
Consider `.pyc` if we end up using Python?

Reply:
Good point.